platform/broadcom/docker-syncd-brcm/Dockerfile.j2

```diff
@@ -27,6 +27,7 @@ RUN chmod +x /usr/bin/dsserve /usr/bin/bcmcmd
 COPY ["supervisord.conf", "/etc/supervisor/conf.d/"]
 COPY ["files/supervisor-proc-exit-listener", "/usr/bin"]
 COPY ["critical_processes", "/etc/supervisor/"]
+COPY ["files/syncd_init_common.sh", "files/syncd_start.sh", "/usr/bin/"]
```

Comment:
Modify per docker image seems changing to many places.
Suggest make the sync/async decision outside the container, even when there is no explicit ConfigDB field/value.